src/test/kotlin/uk/gov/justice/probation/courtcaseservice/client/ManageOffencesRestClientTest.kt

```diff
@@ -0,0 +1,49 @@
+package uk.gov.justice.probation.courtcaseservice.client
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Assertions.assertThrows
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import uk.gov.justice.probation.courtcaseservice.BaseIntTest
+import uk.gov.justice.probation.courtcaseservice.client.exception.ExternalApiEntityNotFoundException
+import uk.gov.justice.probation.courtcaseservice.client.manageoffences.ApiError
+
+class ManageOffencesRestClientTest : BaseIntTest() {
```

Comment:
The project has a convention of IntTest suffix for integration tests. I wondered for a minute, how the tests are working without the mocks 🤭 😆 .. until I saw the base class